### ISSUE TITLE
Unexpected double quotes in Airplay/Shairport device name

### ISSUE THREAD

**Issue by yesman82** [badaix/snapcast#851]
**Describe the bug**
The Airplay device name has double quotes around its name. So in list of airplay devices the Snapcast shairport device is show as _"Snapcast"_ instead of just _Snapcast_.

**Steps to Reproduce**
1. Update your stream variable in your snapserver.conf to `stream = airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊` (same issue with no device name at all)
2. Start/Restart Server
3. Check Apple Device for Airplay devices
4. It shows _"🔊 Multiroom 🔊"_ instead of just _🔊 Multiroom 🔊_

**Environment details**
- OS: Raspbian
- Snapcast version 0.24.0 (not sure if issue appears in this version for the first time)
- Used ARM server (snapserver_0.24.0-1_armhf.deb) from https://github.com/badaix/snapcast/releases/tag/v0.24.0

**Logs**
```log
2021-04-13 09-48-38.598 [Info] (Snapserver) Version 0.24.0, revision 7eff45fd
2021-04-13 09-48-38.598 [Info] (Snapserver) Adding stream: airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊
2021-04-13 09-48-38.598 [Notice] (init) Settings file: "/home/pi/.config/snapserver/server.json"
2021-04-13 09-48-38.611 [Info] (Avahi) Adding service 'Snapcast'
2021-04-13 09-48-38.617 [Debug] (parse) StreamUri: airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊
2021-04-13 09-48-38.618 [Debug] (parse) StreamUri decoded: airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊
2021-04-13 09-48-38.618 [Debug] (parse) scheme: 'airplay' tmp: '///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.618 [Debug] (parse) host: '' tmp: '/shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊' path: '/shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.618 [Debug] (parse) path: '/shairport-sync' tmp: 'name=Airplay&devicename=🔊 Multiroom 🔊' query: 'name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.618 [Debug] (parse) StreamUri.toString: airplay:////shairport-sync?devicename=🔊 Multiroom 🔊&name=Airplay
2021-04-13 09-48-38.618 [Info] (PcmStream) PcmStream: Airplay, sampleFormat: 44100:16:2
2021-04-13 09-48-38.618 [Info] (PcmStream) Stream: Airplay, metadata={
2021-04-13 09-48-38.618 [Info] (PcmStream)     "STREAM": "Airplay"
2021-04-13 09-48-38.618 [Info] (PcmStream) }
2021-04-13 09-48-38.619 [Debug] (Server) metadata = {
2021-04-13 09-48-38.619 [Debug] (Server)    "STREAM": "Airplay"
2021-04-13 09-48-38.619 [Debug] (Server) }
2021-04-13 09-48-38.619 [Info] (Server) onMetaChanged (Airplay)
2021-04-13 09-48-38.619 [Debug] (AsioStream) Chunk duration: 20 ms, frames: 882, size: 3528
2021-04-13 09-48-38.619 [Debug] (ProcessStream) Watchdog timeout: 0
2021-04-13 09-48-38.619 [Info] (Server) Stream: {"fragment":"","host":"","path":"/shairport-sync","query":{"chunk_ms":"20","codec":"ogg","devicename":"🔊 Multiroom 🔊","name":"Airplay","sampleformat":"44100:16:2"},"raw":"airplay:////shairport-sync?chunk_ms=20&codec=ogg&devicename=🔊 Multiroom 🔊&name=Airplay&sampleformat=44100:16:2","scheme":"airplay"}
2021-04-13 09-48-38.620 [Debug] (parse) StreamUri: airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊
2021-04-13 09-48-38.620 [Debug] (parse) StreamUri decoded: airplay:///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊
2021-04-13 09-48-38.620 [Debug] (parse) scheme: 'airplay' tmp: '///shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.620 [Debug] (parse) host: '' tmp: '/shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊' path: '/shairport-sync?name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.620 [Debug] (parse) path: '/shairport-sync' tmp: 'name=Airplay&devicename=🔊 Multiroom 🔊' query: 'name=Airplay&devicename=🔊 Multiroom 🔊'
2021-04-13 09-48-38.620 [Debug] (parse) StreamUri.toString: airplay:////shairport-sync?devicename=🔊 Multiroom 🔊&name=Airplay
2021-04-13 09-48-38.620 [Debug] (PcmStream) Start: Airplay, sampleformat: 44100:16:2
2021-04-13 09-48-38.671 [Debug] (OggEnc) HeadLen: 28, bodyLen: 30, result: 1
2021-04-13 09-48-38.672 [Debug] (OggEnc) HeadLen: 44, bodyLen: 4043, result: 1
2021-04-13 09-48-38.683 [Debug] (ProcessStream) Launching: '/usr/local/bin/shairport-sync', with params: '--name="🔊 Multiroom 🔊" --output=stdout --use-stderr --get-coverart "--metadata-pipename=/tmp/shairmeta.10859.5000" --port=5000', in path: '/usr/local/bin/'
```

![image](https://user-images.githubusercontent.com/550213/114516754-fdcde780-9c3d-11eb-8182-b2585567a06e.png)

**Comment by kingosticks** [comment]
Looks like it used to handle this in https://github.com/badaix/snapcast/commit/c6cbc24163b17ccbbaec46d53055b8901265a18e and then was changed in a big commit (maybe intentionally, maybe not) at https://github.com/badaix/snapcast/commit/ae16e74f6087abe75b842db33f7ab4813615d31b

**Comment by yesman82** [comment]
@badaix Snapshot approved! Double quotes gone. 👍

**Comment by badaix** [comment]
Great, thanks for the confirmation

**Comment by badaix** [comment]
Released in [Snapcast v0.25.0](https://github.com/badaix/snapcast/releases/tag/v0.25.0)
